### ISSUE TITLE
Are newer versions of GHC supported?

### ISSUE THREAD

**Issue by Wizek** [jfischoff/twitch#27]
E.g. on 8.6.1 I get this:

```hs
    Configuring twitch-0.1.7.1...
    Preprocessing library for twitch-0.1.7.1..
    Building library for twitch-0.1.7.1..
    [1 of 7] Compiling Twitch.Path      ( src/Twitch/Path.hs, .stack-work/dist/x86_64-linux/Cabal-2.2.0.1/build/Twitch/Path.o )
    [2 of 7] Compiling Twitch.Rule      ( src/Twitch/Rule.hs, .stack-work/dist/x86_64-linux/Cabal-2.2.0.1/build/Twitch/Rule.o )
    [3 of 7] Compiling Twitch.InternalRule ( src/Twitch/InternalRule.hs, .stack-work/dist/x86_64-linux/Cabal-2.2.0.1/build/Twitch/InternalRule.o )
    
    /tmp/stack12594/twitch-0.1.7.1/src/Twitch/InternalRule.hs:101:3: error:
        • The constructor ‘Added’ should have 3 arguments, but has been given 2
        • In the pattern: Added file tyme
          In a case alternative: Added file tyme -> modify rule file tyme
          In the expression:
            case event of
              Added file tyme -> modify rule file tyme
              Modified file tyme -> add rule file tyme
              Removed file tyme -> delete rule file tyme
        |
    101 |   Added    file tyme -> modify rule file tyme
        |   ^^^^^^^^^^^^^^^^^^
```

**Comment by menelaos** [comment]
I can't check right now, but the `master` branch should work.

**Comment by menelaos** [comment]
I've published the current version as `v0.1.7.2` on Hackage.
